index.js

```diff
@@ -219,6 +219,23 @@ function focusTrap(element, userOptions) {
     return node;
   }
 
+  function getReturnFocusNode(previousActiveElement) {
+    var node;
+    if (getNodeForOption('setReturnFocus') !== null) {
```

Comment:
Can we extract `getNodeForOption('setReturnFocus')` to a variable in a line before it, so the `getNodeForOption` is called only once, not twice, in case it returns something?

Reply:
Refactored it.